packages/langium/src/parser/async-parser.ts

```diff
@@ -27,15 +27,15 @@ export interface AsyncParser {
      * @param cancelToken A cancellation token that can be used to cancel the parsing process.
      * @returns A promise that resolves to the parse result.
      *
-     * @throw `OperationCancelled` if the parsing process is cancelled.
+     * @throws `OperationCancelled` if the parsing process is cancelled.
      */
     parse<T extends AstNode>(text: string, cancelToken: CancellationToken): Promise<ParseResult<T>>;
 }
 
 /**
  * Default implementation of the async parser which simply wraps the sync parser in a promise.
  *
- * @remark A real implementation would create worker threads or web workers to offload the parsing work.
```

Comment:
Same here as above. If it's not an issue with TypeDoc it would be good to retain the remarks as they were.

Reply:
It's just a warning, so we can ignore it, but I just didn't see a reason for it since it is not a valid JSDoc tag